compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/scene/WindowComposeSceneLayer.desktop.kt

```diff
@@ -187,8 +208,20 @@ internal class WindowComposeSceneLayer(
         )
     }
 
+    override fun onRenderOverlay(canvas: Canvas, width: Int, height: Int) {
+        val paint = scrimColor?.let { scrimColor ->
```

Comment:
_nit:_

It is not easy to read because of nested let/apply. I would suggest this:
```
val scrimColor = scrimColor ?: return
val paint = Paint().apply { color = scrimColor }.asFrameworkPaint()
```